demo/index.html

```diff
@@ -25,6 +25,13 @@
           const rawHtml = marked.parse(text);
           document.querySelector('main').innerHTML = rawHtml;
           Prism.highlightAll();
+        })
+        .then(() => {
+          const targetButton = document.getElementById('targetButton');
```

Comment:
We need to use better selectors here. `targetButton` is very generic and could be inadvertently used elsewhere on the page. 

Also, this approach needs to be updated to use the new JS example method so it will work on the docsite.